requirements.txt

```diff
@@ -1,8 +1,9 @@
-dwave-ocean-sdk==2.6.0
+dwave-ocean-sdk>=2.6.0
```

Comment:
For guaranteed QPU selection by DWaveSampler, this has to be `>=3.0.0`.

Reply:
Updated to >=3.0.0, thanks